datasets/brandkranen/dataset.json

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "brandkranen",
+    "type": "dataset",
+    "auth": "FP/MDW",
```

Comment:
In de omschrijving staat dat deze dataset de openbare brandkranen laat zien, maar de dataset zelf is alleen beschikbaar voor medewerkers. Klopt het dat deze dataset niet volledig open is?

Reply:
goed punt. het gaat om de brandkranen in de openbare ruimte. Ik pas de beschrijving aan.